CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.39.0 (2021-04-20)
+* Adds `syncPagesMoves` flag to move live pages at the same time than draft ones, without committing.
```

Comment:
Let's go with `autoCommitPageMoves` to clarify what this does.

Reply:
Hum actually this is not a commit so I'm not sure it clarifies..

Reply:
I modified it anyway, it's a part of a commit you're right.